clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3967,6 +3967,80 @@ Attribute ``trivial_abi`` has no effect in the following cases:
   }];
 }
 
+
+def LifetimeCaptureByDocs : Documentation {
```

Comment:
nit: let's move this section immediately right after the `LifetimeboundDocs`.

Reply:
Done.